boosters/node-beginner/tests/03_exercise.test.js

```diff
@@ -0,0 +1,20 @@
+const request = require("supertest");
+const app = require("../src/app");
+const teardown = require("./config/teardown");
+const uuid = require("uuid/v4");
+
+const user = {
+  firstName: "Andrew",
+  lastName: "Mayer",
+  id: uuid(),
```

Comment:
Doesn't the `id` field get generated by sequelize already?

Reply:
It does for 'create'. I'm using 'bulk create'. We don't have a hook for that. The reason I'm creating it here  is that usually after creating the user, I need to UUID to test it